src/components/Detail/index.js

```diff
@@ -14,10 +15,11 @@ const Detail = () => {
   }, [dispatch]);
 
   const hanldeBack = () => {
-    navigate('/');
+    navigate('/home');
```

Comment:
the back button show return to root '/' there is the home, please change this to navigate('/');

Reply:
This was probably some merge issue, I already changed the url properly